components/Dashboard/Post/PostComment.tsx

```diff
@@ -79,7 +84,9 @@ const PostComment: React.FC<PostCommentProps> = ({
                 ? `${comment.author.name} (@${comment.author.handle})`
                 : `@${comment.author.handle}`}
             </span>
-            <span className="comment-date">{formatDateRelativeToNow(comment.createdAt)} ago</span>
+            <span className="comment-date">
+              {formatDateRelativeToNow(comment.createdAt)} {t('relativeTimeWord')}
+            </span>
```

Comment:
@Lanny I realized here that currently our little time library function is what returns "minutes" or "hours" or "days", etc... for the relative time comparison, and the position of the equivalent of "ago" will change based on the language 🧐